.travis.yml

```diff
@@ -4,4 +4,4 @@ sudo: required
 dist: trusty
 before_install: ./maintainers/scripts/travis-nox-review-pr.sh nix
 install: ./maintainers/scripts/travis-nox-review-pr.sh nox
-script: ./maintainers/scripts/travis-nox-review-pr.sh build
+script: travis_wait 119 ./maintainers/scripts/travis-nox-review-pr.sh build
```

Comment:
Yes, but I expect travis to let that print happen. Otherwise we can modify that function to do `tail -c 100k` for the print